### ISSUE TITLE
[BUG] Action log not working

### ISSUE THREAD

**Issue by dimiandre** [Maatwebsite/Laravel-Nova-Excel#45]
### Prerequisites
Activate the action log following nova documentation https://nova.laravel.com/docs/1.0/actions/defining-actions.html#action-log 
(adding **use Actionable** to the model that refers to resource that has ExportToExcel or DownloadExcel action) 

* [x] Able to reproduce the behaviour outside of your code, the problem is isolated to Laravel Excel.
* [x] Checked that your issue isn't already filed.
* [x] Checked if no PR was submitted that fixes this problem.

### Versions
* PHP version: 7.2.1
* Laravel version: 5.8
* Nova version: 2.0
* Package version: 1.1

### Description
If you activate the Action log, when you export something you should see the log inside the single view of the resource. This does not happen. I tried other actions and those are being logged correctly.

### Steps to Reproduce

1. Activate the action log following nova documentation https://nova.laravel.com/docs/1.0/actions/defining-actions.html#action-log 
2. Add ExportToExcel or DownloadExcel action to your resource
3. Do an export

**Expected behavior:**

Create a new log of the performed action

**Actual behavior:** 

Nothing is created

**Comment by patrickbrouwers** [comment]
Hey @dimiandre . What do you expect to end up in the action log. That the resource has been exported or that someone started an export?

**Comment by dimiandre** [comment]
> Hey @dimiandre . What do you expect to end up in the action log. That the resource has been exported or that someone started an export?

@patrickbrouwers thanks for the reply :) 
I think in case of DownloadExcel action would be nice to have a log of the resource has been exported
and in case of ExportToExcel would be nice to have a log of the resource has started exporting

what do you think?

**Comment by patrickbrouwers** [comment]
So you mean for each resource (example each user) that is in the export, have a log entry ?

**Comment by dimiandre** [comment]
> So you mean for each resource (example each user) that is in the export, have a log entry ?

yes! but as i said, it's a native feature of nova the "action log". 
It should work by default each time you create a cutom action, I don't know why it's not working with this action

**Comment by patrickbrouwers** [comment]
It's because the export action is not like any other nova action, it doesn't inherit all the functionality because we delegate the querying (for performance reasons) to the laravel excel package. 
I didn't enable the action log, because it didn't seem valuable to me. 
I'm fine with adding it, however it's not really high on my current prio-list. If it's of importance to you (to have it prioritize) you can check our options for commercial support: https://laravel-excel.com/commercial-support

**Comment by dimiandre** [comment]
ok! 

thanks a lot :)
